README.md

```diff
@@ -66,37 +69,39 @@ php artisan translatable:inspect-translations fr --export-first
 
 ### Find untranslated strings in a language file (IDE)
 
-An alternative way to find untranslated strings in your language files is to search for entries with the same string for original and translated. 
+An alternative way to find untranslated strings in your language files is to search for entries with the same string for original and translated.
 You can do this in most editors using a regular expression.
 
-In PhpStorm, you can use this pattern: `"([^"]*)": "\1"`
+In PhpStorm and VSCode, you can use this pattern: `"([^"]*)": "\1"`
```

Comment:
Did you test the regular expression in VSCode?

Reply:
Yes, it was tested in vscode and it works correctly.

![image](https://user-images.githubusercontent.com/13445515/212071963-ba16b2f8-172c-4663-a9cb-68cbe2823fb1.png)

Reply:
Great. Thanks